### ISSUE TITLE
Windows 10 mp4 files dont play

### ISSUE THREAD

**Issue by UTSAGrad** [Sascha-L/WPF-MediaKit#16]
I have successfully downloaded this library and have run the sample application on windows 7 and it can read avi, wmv, and mp4 just fine.  I repeat this process on a windows 10 pc 64bit and only avi and wmv videos render.  It cannot play the MP4 videos on windows 10.  I tried it on a 2nd windows 10 64it pc with the same results.  I can play mp4 videos with MS Media Player on all these pc's.  All are using Visual Studio 2015 pro.  "Help me Obi Wan Kanobi.  You're my only hope."

**Comment by skaterlui** [comment]
Hello,
it was the same for me. 
I installed k-lite-codec pack and switched Playback h.264 to fddshow. Then it worked on Windows 10 64bit.

**Comment by magicman86** [comment]
I had to install the LAV splitter to get it to work

**Comment by UTSAGrad** [comment]
Thank you very much for your help.  I was able to get it to work on Windows 10 after I installed the K-Lite mega codec pack.  Select the expert installation option and then on the select components screen there is the option DirectShow video decoding filters section, within the H.264 option, select the ffdshow item.  Accept all other default options on all the rest of the screens presented to install.
